assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -215,7 +216,11 @@ export default function WidgetAreaRenderer( { slug, contextID } ) {
 								<div className="googlesitekit-widget-area-header__details">
 									{ subtitle && (
 										<h4 className="googlesitekit-widget-area-header__subtitle">
-											{ subtitle }
+											{ typeof subtitle === 'function' ? (
+												<widgetArea.subtitle />
+											) : (
```

Comment:
To adapt to the new design of conditionally displaying the subtitle, had to make small modification to widget logic, to make subtitle more flexible - allowing it to render component when passed